### PR TITLE
Adds annotation

### PR DESCRIPTION
**Pull request by DhritiShikhar**
Adds annotation to deployment and service config.

TODO:

- [x] Add annotation to route config after https://github.com/redhat-developer/topology-example/pull/5 PR is merged

### CODE REVIEW

File: application-1/analytics/deployment.yaml [hunk 1]
```diff
@@ -8,6 +8,9 @@ metadata:
     app.kubernetes.io/component: backend
     app.kubernetes.io/instance: user-analytics
     app.kubernetes.io/version: "1.0"
+  annotations:
+    app.openshift.io/vcs-uri: "git@github.com:redhat-developer/topology-example.git"
```
Comment (sbose78) [inline application-1/analytics/deployment.yaml:12]: Also, do open a PR to fix the typo on app-labels repo, @DhritiShikhar .

File: application-1/analytics/deployment.yaml [hunk 2]
```diff
@@ -8,6 +8,9 @@ metadata:
...
+    app.openshift.io/vcs-ref: "v1.0.0"
```
Comment (sbose78) [inline application-1/analytics/deployment.yaml]: Could you add a different ref ? Something more realistic?

**Review by sbose78 — CHANGES_REQUESTED** [review]

File: application-1/analytics/deployment.yaml (hunk 1 above)
Comment (DhritiShikhar) [reply application-1/analytics/deployment.yaml:12]: @sbose78 are you talking about this - https://github.com/gorkem/app-labels/pull/11 ?

File: application-1/analytics/deployment.yaml (hunk 2 above)
Comment (DhritiShikhar) [reply application-1/analytics/deployment.yaml]: I have put commit hash - 
d1e1433

**Review by sbose78 — APPROVED** [review]

**Pull request merged**